BismarkUnchained/scenes/KinematicBody2D.gd

```diff
@@ -1,5 +1,29 @@
 extends KinematicBody2D
 
-func _physics_process(delta):
-	move_and_collide(Vector2(50, 0))
+var velocity
```

Comment:
can be moved into `_process`, depending on what you're doing for controller input